bin/export.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-from __future__ import print_funcion
+from __future__ import print_function
 import fitsio
```

Comment:
This fix was already implemented on master, I think your master is not up to date. Could you 
    git checkout master
    git pull
    git checkout add_unitest_fitter2
    git merge master
    git push origin add_unitests_fitter2